com.unity.ml-agents/Runtime/Policies/RemotePolicy.cs

```diff
@@ -33,6 +36,15 @@ internal class RemotePolicy : IPolicy
         /// <inheritdoc />
         public void RequestDecision(AgentInfo info, List<ISensor> sensors)
         {
+            if (!m_AnalyticsSent)
+            {
+                m_AnalyticsSent = true;
+                TrainingAnalytics.RemotePolicyStarted(
+                    m_FullyQualifiedBehaviorName,
+                    sensors,
+                    m_ActionSpec
+                );
+            }
```

Comment:
I think these lines should live inside a method in TrainingAnalytics.cs.
In this script, this should be a single method call like TrySendPolicyStartedAnalyticsEvent(m_FullyQualifiedBehaviorName, sensors, m_ActionSpec)

Reply:
This is doing what we currently do in BarracudaPolicy. I thought there was feedback on the inference PR (although I can't find it now) that we should make explicit that we send at most one event per policy (since this is happening in RequestDecision). There's additional deduping in the TrainingAnalytics code, so it's not strictly necessary, we should just be consistent.